js/scripts/generate-wrappers.js

```diff
@@ -184,38 +219,66 @@ function mapPromiseFnOverFileList(fileList, mapFn) {
 }
 
 function mapPromiseFnOverThreeModules(mapFn) {
-    return mapPromiseFnOverGlob('**/*.js', mapFn, { 
-        cwd: threeSrcDir, 
+    return mapPromiseFnOverGlob('**/*.js', mapFn, {
+        cwd: threeSrcDir,
         nodir: true ,
         ignore: [
             '**/Three.Legacy.js',
+            '**/polyfills.js',
```

Comment:
might be nice to add some comments as to why these files are ignored, so that, in the future, if something changes, we can remove some of the files if we end up supporting them